fdbserver/tester.actor.cpp

```diff
@@ -1473,6 +1490,18 @@ std::map<std::string, std::function<void(const std::string& value, TestSpec* spe
 	  } },
 	{ "runFailureWorkloads",
 	  [](const std::string& value, TestSpec* spec) { spec->runFailureWorkloads = (value == "true"); } },
+	{ "disabledFailureInjectionWorkloads",
+	  [](const std::string& value, TestSpec* spec) {
+	      std::stringstream ss(value);
```

Comment:
I am not quite sure if this would be a simple modification, but since TOML has array type, it might be more reasonable to allow the user write

```
disabledFaultInjectionWorkloads = ["Attrition"]
```

and let the TOML parser do the job, rather use this mini-parser?

Reply:
This is because for TOML file, the tester converts values into string here https://github.com/apple/foundationdb/blob/4641c9b2a7a42147ac2e9e499aeb618594ef8f29/fdbserver/tester.actor.cpp#L1647. So we need the parsing anyway.

Reply:
This is indeed against intuition.

Reply:
Added a comment on why the custom encoding is needed and explain this expects a comma separated list.